app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -155,16 +155,15 @@ class TopicLessonsFragmentTest {
   @Inject
   lateinit var explorationCheckpointTestHelper: ExplorationCheckpointTestHelper
 
-  @JvmField
-  @field:[Inject EnablePracticeTab]
-  var enablePracticeTab: Boolean = false
+  var enableExtraTopicTabsUiValue: Boolean = false
```

Comment:
Shouldn't this be replaced with an injection of the new platform parameter? Otherwise it's never really effectively changing.

Ditto in TopicTestActivityForStoryTest and TopicPracticeFragmentTest.

Reply:
Yes, the value for the parameter is changed via function enableExtraTopicTabsUi(). Done.

Reply:
I think this is still unresolved, but I added a new comment--PTAL.